files/ocs-ci/ocs-ci-05-addcapacity-skip.patch

```diff
@@ -0,0 +1,141 @@
+ from ocs_ci.ocs import defaults
+@@ -31,6 +35,7 @@ logger = logging.getLogger(__name__)
+ )
+ @ignore_leftovers
+ @tier2
++@skipif_ibm_power
+ @pytest.mark.skipif(
+     config.ENV_DATA["platform"].lower() == constants.VSPHERE_PLATFORM,
+     reason=(
+diff --git a/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py b/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
+index 855c4a8e..ab93411d 100644
+--- a/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
++++ b/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
+@@ -1,6 +1,11 @@
+ import pytest
+ import logging
+ 
++from ocs_ci.framework.pytest_customization.marks import (
++    skipif_ibm_power,
++)
++
++
+ from ocs_ci.framework.testlib import ignore_leftovers, ManageTest, tier4a
+ from ocs_ci.framework import config
+ from ocs_ci.ocs import constants
+@@ -23,6 +28,7 @@ from ocs_ci.ocs.resources.storage_cluster import osd_encryption_verification
+ )
+ @ignore_leftovers
+ @tier4a
++@skipif_ibm_power
+ class TestAddCapacityNodeRestart(ManageTest):
+     """
+     Test add capacity when one of the worker nodes got restart
+diff --git a/tests/manage/z_cluster/cluster_expansion/test_delete_pod.py b/tests/manage/z_cluster/cluster_expansion/test_delete_pod.py
+index 5612e5f2..e96413ed 100644
+--- a/tests/manage/z_cluster/cluster_expansion/test_delete_pod.py
++++ b/tests/manage/z_cluster/cluster_expansion/test_delete_pod.py
+@@ -3,6 +3,11 @@ import pytest
+ import logging
+ from concurrent.futures import ThreadPoolExecutor
+ 
++from ocs_ci.framework.pytest_customization.marks import (
++    skipif_ibm_power,
++)
++
++
+ from ocs_ci.framework.testlib import ignore_leftovers, tier4a
+ from ocs_ci.framework import config
+ from ocs_ci.ocs import constants
+@@ -16,6 +21,7 @@ from ocs_ci.helpers.disruption_helpers import Disruptions
+ 
+ @ignore_leftovers
+ @tier4a
++@skipif_ibm_power
+ class TestAddCapacityWithResourceDelete:
+     """
+     Test add capacity when one of the resources gets deleted
+diff --git a/tests/manage/z_cluster/cluster_expansion/test_node_expansion.py b/tests/manage/z_cluster/cluster_expansion/test_node_expansion.py
+index cb5b667c..1f0d8e1b 100644
+--- a/tests/manage/z_cluster/cluster_expansion/test_node_expansion.py
++++ b/tests/manage/z_cluster/cluster_expansion/test_node_expansion.py
+@@ -1,8 +1,12 @@
+ import logging
+ 
++from ocs_ci.framework.pytest_customization.marks import (
++    skipif_openshift_dedicated,
++    skipif_ibm_power,
++)
++
+ from ocs_ci.framework.testlib import tier1, ignore_leftovers, ManageTest
+ from ocs_ci.ocs.cluster import CephCluster
+-from ocs_ci.framework.pytest_customization.marks import skipif_openshift_dedicated
+ 
+ logger = logging.getLogger(__name__)
+ 
+@@ -10,6 +14,7 @@ logger = logging.getLogger(__name__)
+ @skipif_openshift_dedicated
+ @ignore_leftovers
+ @tier1
++@skipif_ibm_power
+ class TestAddNode(ManageTest):
```

Comment:
We support add node on powervs.   Does this test fail?   Does the test do something else like add capacity that is not supported?